nemo/core/classes/common.py

```diff
@@ -585,7 +571,7 @@ def to_config_file(self, path2yaml_file: str):
         """
         if hasattr(self, '_cfg'):
             self._cfg = maybe_update_config_version(self._cfg)
-            with open(path2yaml_file, 'w', encoding='utf-8') as fout:
+            with open(path2yaml_file, 'w') as fout:
```

Comment:
This is required. It causes encoding errors across all Nemo models.

Reply:
Done.